1.6/run.sh

```diff
@@ -24,7 +24,7 @@ fi
 
 # if `docker run` first argument start with `--` the user is passing jenkins launcher arguments
 if [[ $# -lt 1 ]] || [[ "$1" == "--"* ]]; then
-   exec java $JAVA_OPTS -jar /usr/lib/jenkins/jenkins.war $JENKINS_OPTS "$@"
+   exec java $JAVA_OPTS -Dfile.encoding=UTF8 -jar /usr/lib/jenkins/jenkins.war $JENKINS_OPTS "$@"
```

Comment:
fixes this:
https://bugzilla.redhat.com/show_bug.cgi?id=1247894
see:
https://stackoverflow.com/questions/27960996/jenkins-console-output-characters